pom.xml

```diff
@@ -87,6 +87,12 @@
 	</repositories>
 
 	<dependencies>
+		<dependency>
+			<groupId>com.aliyun.oss</groupId>
+			<artifactId>aliyun-sdk-oss</artifactId>
+			<version>2.8.3</version>
+		</dependency>
```

Comment:
Could this and AWS dependencies be moved to profiles so that they are not loaded unless the profile is loaded.